### PR TITLE
Add example for non-native 3rd party package installation

### PR DESCRIPTION
**Pull request by Definitely-Not-Vlad**
Adding an example extension that uses react-native-swiper.

This is the result of:
1) Installing a non-native Package

From the "Installing 3rd Party Packages" tutorial.

### CODE REVIEW

**Comment by tenodi** [comment]
Setup eslint and run through your examples. Once there are no errors, tag me to check it out again.

**Comment by Definitely-Not-Vlad** [comment]
@tenodi Ran ESLint as requested.

**Comment by tenodi** [comment]
Same comment, ignore `.DS_Store` globally.

File: swiper-extension/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+**/build/*
```
Comment (tenodi) [inline swiper-extension/.eslintignore]: Use the repo's `.eslintignore`.

File: swiper-extension/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (tenodi) [inline swiper-extension/.eslintrc]: Use the repo's `.eslintrc`.

**Comment by tenodi** [comment]
Make sure `node_modules` are ignored as well. This can be done on the repo level too. You've submitted `node_modules` in this PR.

File: swiper-extension/app/package.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "vladimir-vdovic.swiper-extension",
```
Comment (tenodi) [inline swiper-extension/app/package.json]: `tom` or `vlad`.

**Review by tenodi — CHANGES_REQUESTED** [review]
This is good! Just a few things to be changed.

**Comment by Definitely-Not-Vlad** [comment]
@tenodi
Should be ready for merge.

No .DS_Store files.

No node_modules folder.

ESLint ran.

Requested changes done.

**Pull request closed without merging**